include/drivers/console.h

```diff
@@ -7,14 +7,59 @@
 #ifndef __CONSOLE_H__
 #define __CONSOLE_H__
 
-#include <stdint.h>
+#include <utils_def.h>
 
-int console_init(uintptr_t base_addr,
-		unsigned int uart_clk, unsigned int baud_rate);
+#ifdef AARCH32
+#define REGSZ U(4)
+#else
+#define REGSZ U(8)
```

Comment:
minor: I hoped stdlib would have something you can use here but I can't see anything quite right. Perhaps this could be moved to utils_def.h (and called REG_SIZE)?

Reply:
Yeah, there are only a few independent implementations of this scattered around, like in crash_reporting.S. I can consolidate them into utils_def.h.